Automattic/wp-calypso
# Readymade Templates: Update the code to use the new API format

Related to #https://github.com/Automattic/dotcom-forge/issues/8137

## Proposed Changes

* Update the Readymade Templates onboarding to use the new API format
* Update the /patterns page to use the new pattern format (for the copy button)
* Hardcode the image to Neofit, for now, until the block reviewer is in place

## Why are these changes being made?


* The new format supports template parts on target sites

## Testing Instructions


* Apply D155039-code on your sandbox
* Use the live link and go to /patterns
* You'll notice a single RMT (T-Rex 2)
* Notice that the image is hardcoded to Neofit
* Click on it and make sure the copy button works correctly (the clipboard contains the full RMT pattern)
* Click on the Pick layout button and complete the Onboarding
* Notice that the new site you've created now contains a header template part, a content and the footer template part

## Pre-merge Checklist
